backend/app/assets/javascripts/spree/backend.js

```diff
@@ -13,6 +13,7 @@
 //= require spree/backend/spree-select2
 //= require_tree .
 
+Spree.routes.clear_cache = Spree.pathFor('admin/general_settings/clear_cache')
```

Comment:
Identifier 'clear_cache' is not in camel case.<br>Missing semicolon.<br>'Spree' is not defined.